src/MahApps.Metro/Styles/VS/Window.xaml

```diff
@@ -32,4 +33,43 @@
         <Setter Property="LightMinButtonStyle" Value="{StaticResource VSWindowButtonStyle}" />
     </Style>
 
+    <SolidColorBrush x:Key="NotificationButtonForegroundBrush" Color="#000000" />
+    <SolidColorBrush x:Key="NotificationButtonBackgroundBrush" Color="#FFCC00" />
+    <SolidColorBrush x:Key="NotificationButtonBackgroundOverBrush" Color="#FFDF66" />
+    <SolidColorBrush x:Key="NotificationButtonBackgroundPressedBrush" Color="#C59E00" />
+
+    <SolidColorBrush x:Key="QuickLaunchTextBoxBorderBrush" Color="#3F3F46" />
+    <SolidColorBrush x:Key="QuickLaunchTextBoxFocusedBorderBrush" Color="#007ACC" />
+
```

Comment:
@swdriessen Please move these brushes to the [Colors.xaml](https://github.com/MahApps/MahApps.Metro/blob/develop/src/MahApps.Metro/Styles/VS/Colors.xaml)

Reply:
Done